pkg/vm/engine/tae/catalog/metabase.go

```diff
@@ -60,7 +60,37 @@ func (be *MetaBaseEntry) PPString(level common.PPLevel, depth int, prefix string
 	s := fmt.Sprintf("%s%s%s", common.RepeatStr("\t", depth), prefix, be.StringLocked())
 	return s
 }
+func (be *MetaBaseEntry) GetMetaLoc() string {
+	be.RLock()
+	defer be.RUnlock()
+	if be.GetNodeLocked() == nil {
+		return ""
+	}
+	str := be.GetNodeLocked().(*MetadataMVCCNode).MetaLoc
+	return str
+}
+func (be *MetaBaseEntry) GetDeltaLoc() string {
```

Comment:
Currently, there may be no problem here. But later we should always get an mvcc-value.

Reply:
GetVisibleDeltaLoc can get mvcc-value by timestamp